language-plutus-core/generators/Language/PlutusCore/Generators/Internal/TypeEvalCheck.hs

```diff
@@ -105,8 +105,8 @@ typeEvalCheckBy eval (TermOf term x) = TermOf term <$> do
 -- | Type check and evaluate a term and check that the expected result is equal to the actual one.
 -- Throw an error in case something goes wrong.
 unsafeTypeEvalCheck
-    :: ( KnownType uni a, GShow uni, GEq uni, DefaultUni <: uni, Closed uni
-       , uni `Everywhere` Eq, uni `Everywhere` Pretty, uni `Everywhere` ExMemoryUsage
+    :: ( KnownType uni a, GShow uni, GEq uni, DefaultUni <: uni, Closed uni, uni `Everywhere` Eq
+       , uni `Everywhere` PrettyConst, uni `Everywhere` ExMemoryUsage
```

Comment:
Oops, forgot to refactor that. Please make it ``uni `EverywhereAll` '[Eq, PrettyConst, ExMemoryUsage]``.